### PR TITLE
#138: Removed Exasol Artifactory references from user guide

### LINKED ISSUE #138: Update user guide
### Situation

The user guide have several references to the Exasol maven artifactory. However, we migrated to Maven central in recent version. We should remove the references to artifactory from user guide.

### Acceptance Criteria

- Remove Exasol artifactory references

### PR DESCRIPTION
**Pull request by morazow**
Fixes #138

### CODE REVIEW

File: doc/user_guide/user_guide.md [hunk 1]
```diff
@@ -78,13 +62,10 @@ To provide the connector to your Spark Scala projects, add the following lines
...
 
 ```scala
-resolvers ++= Seq("Exasol Releases" at "https://maven.exasol.com/artifactory/exasol-releases")
-
 libraryDependencies += "com.exasol" % "spark-connector" %% "<VERSION>"
 ```
 
-Similar to the Java dependency, we add the resolver to the Exasol Artifactory so
-that the Exasol JDBC driver can be found.
+Similar please do not forget to update the `<VERSION>` placeholder.
```
Comment (redcatbear) [inline doc/user_guide/user_guide.md:68]:
```suggestion
Similarly, please do not forget to update the `<VERSION>` placeholder.
```

File: doc/user_guide/user_guide.md [hunk 2]
```diff
@@ -97,27 +78,22 @@ Similar to using maven, you should provide maven artifact coordinates to the
...
-  Scala version 2.12, change it to 2.13 if your Databricks Runtime version
-  requires it.
-- In the Repository field, enter the Exasol Artifactory
-  `https://maven.exasol.com/artifactory/exasol-releases`.
-- Click Install.
+- Select Maven as a `Library Source`
+- In the `Coordinates` field, enter artifact coordinates
+  `com.exasol:spark-connector_2.13:<VERSION>`. Please notice that we use the
+  Scala version 2.13, please let us know if you require Scala version 2.12.
+- Click `Install`
```
Comment (redcatbear) [inline doc/user_guide/user_guide.md:85]:
```suggestion
- In the `Coordinates` field, enter artifact coordinates
  `com.exasol:spark-connector_2.13:<VERSION>`. Please note that we use the
  Scala version 2.13. Please contact us know if you require Scala version 2.12.
- Click `Install`
```

File: doc/user_guide/user_guide.md [hunk 3]
```diff
@@ -133,13 +109,12 @@ Use the `spark-submit` command:
...
-  --repositories https://maven.exasol.com/artifactory/exasol-releases \
-  --packages com.exasol:spark-connector_2.11:<VERSION> \
+  --packages com.exasol:spark-connector_2.13:<VERSION> \
   --class com.organization.SparkApplication \
   path/to/spark_application.jar
 ```
 
-The `--repositories` and `--packages` can be omitted if your Spark application
+The `--packages` parameter can be omitted if your Spark application
 JAR already includes the connector as a dependency (e.g, jar-with-dependencies).
```
Comment (redcatbear) [inline doc/user_guide/user_guide.md:118]:
```suggestion
JAR already includes the connector as a dependency (e.g, `jar-with-dependencies`).
```

File: doc/user_guide/user_guide.md [hunk 4]
```diff
@@ -172,13 +147,13 @@ mvn package -DskipTests=true
 The assembled jar file with the `-assembly` suffix should be located in the `target/`
 folder.
 
-If you want different version of Spark, you can use profiles `-Pspark3.1` or
-`-Pspark2.4` for Spark `3.1` or `2.4` versions respectively.
+If you want different version of Spark, you can use profiles `-Pspark3.2` or
+`-Pspark3.1` for Spark `3.2` or `3.1` versions respectively.
```
Comment (redcatbear) [inline doc/user_guide/user_guide.md:151]:
```suggestion
`-Pspark3.1` for Spark `3.2` or `3.1` versions, respectively.
```
See https://linguaholic.com/linguablog/comma-before-respectively/

File: pom.xml [hunk 5]
```diff
@@ -2,18 +2,20 @@
...
     <parent>
         <artifactId>spark-connector-generated-parent</artifactId>
         <groupId>com.exasol</groupId>
-        <version>1.2.4</version>
+        <version>1.2.5</version>
         <relativePath>pk_generated_parent.pom</relativePath>
     </parent>
     <properties>
         <log4j.version>2.20.0</log4j.version>
+        <sonar.sources>.</sonar.sources>
```
Comment (redcatbear) [inline pom.xml:17]: I don't see those two new properties being used in the file?
What is their purpose here?

Maybe a comment would help.

**Review by redcatbear — CHANGES_REQUESTED** [review]
Couple of small questions and suggestions.

File: doc/user_guide/user_guide.md (hunk 4 above)
Comment (morazow) [reply doc/user_guide/user_guide.md:151]: Nice, didn't know it :))

File: pom.xml (hunk 5 above)
Comment (morazow) [reply pom.xml:17]: Yes, it is mainly for Sonar, since it doesn't find source files `src/scala` and coverage is not reported. I am going to add comment.

Comment (redcatbear) [reply pom.xml:18]: Great, thanks.

**Review by redcatbear — APPROVED** [review]

**Pull request merged**